### PR TITLE
Cr 653 - cuc tests for telephone capture

### PR DESCRIPTION
**Pull request by cookel2**
# Motivation and Context

This  change is required in order to meet the following acceptance test requirements:
- CR-T144: Launch EQ for a household
- CR-T145: Launch EQ for an individual 

Launch EQ for an Individual
# What has changed

The following feature has been created:
- testTelephoneEndpoint

The testTelephoneEndpoint feature contains the following two scenarios:

Scenario: Launch EQ for a household
Scenario: Launch EQ for an Individual

# How to test?

Test the changes by running all the census-contact-centre-cucumber features. To do this the following services need to be running:-

- contact-centre-service 
- census-mock-case-api-service

Once these are running then you can run all the features using this command:

mvn test

### CODE REVIEW

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java [hunk 1]
```diff
@@ -229,6 +251,370 @@ public void no_cases_for_my_UPRN_are_returned(String httpError) {
...
+    hashKeysExpected.add("survey");
+    hashKeysExpected.add("exp");
+    hashKeysExpected.add("period_id");
+    hashKeysExpected.add("iat");
+    hashKeysExpected.add("jti");
+    hashKeysExpected.add("region_code");
+
+    log.info("The hash keys expected are: " + hashKeysExpected.toString());
+
+    List<String> hashKeysFound = new ArrayList<>(result1.keySet());
```
Comment (andrewjohnysons) [inline src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java]: You can also use Arrays.asList(result1.keySet()) - for this purpose.
Not a must, but it's useful to use.

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java [hunk 2]
```diff
@@ -272,4 +658,44 @@ private HttpStatus checkMockCaseApiRunning() {
 
     return mockCaseApiResponse.getStatusCode();
   }
+
```
Comment (andrewjohnysons) [inline src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java]: Do we need the following commented out code?
It's not clean when we do that. :-)

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java (hunk 1 above)
Comment (cookel2) [reply src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java]: OK, I'll bear that in mind, it works ok like this though so I'll leave it for now thanks.

File: src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java (hunk 2 above)
Comment (cookel2) [reply src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java]: good spot Andy - commented code has now been removed

**Pull request merged**